lib/tasks/dell_demo.rake

```diff
@@ -253,7 +253,7 @@ namespace :dell_demo do
           break
         end
 
-        if input.to_i != 0
+        if input.size == 36
```

Comment:
Could we get a little bit of context on what's going on here? A one-sentence PR description would suffice. Good for future-us looking back.

Reply:
Earlier we were using integers as ids and now we moved to using UUIDs. So instead of checking if the input is an integer, we are checking if the input length is 36.